plugins/rapid7_insightidr/help.md

```diff
@@ -1427,15 +1427,17 @@ This action is used to get query results for a LEQL query by query ID.
 
 ##### Input
 
-|Name|Type|Default|Required|Description|Enum|Example|
+|Name|Type| Default |Required| Description |Enum|Example|
```

Comment:
Seems the formatting for this line has now gone off, try and use the comparison git shows to fix it up 😄